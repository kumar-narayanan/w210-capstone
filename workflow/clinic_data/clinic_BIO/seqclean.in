I am looking for a covid shot clinic in the north side of town
How about the south side of town?
Does it offer moderna?
book it for 3 people on tuesday at 11:00 am.
My name is Michael Jackson and number is 7103594862.
No that is all I need. Thank you for your time.
