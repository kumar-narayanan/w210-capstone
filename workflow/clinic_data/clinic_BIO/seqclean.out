O O O O O O O O O O O O B-servicetype O 
O O O O O O O B-clinic-bookperson O B-booktime O B-bookday 
O O O O O O O 
O O O O 
O O O O O O O O O B-covidbrand B-servicetype O 
O O O O O O B-covidbrand 
O O O O O B-area O 
O O O O O B-bookpeople O O B-booktime O B-bookday 
O O O O O O O 
O O O O O 
O O O O O B-servicetype O O O O B-area O O O 
O O O B-area O O O 
O O O B-covidbrand 
O O O B-bookpeople O O B-bookday O B-booktime O 
O O O O O O O O O O O 
