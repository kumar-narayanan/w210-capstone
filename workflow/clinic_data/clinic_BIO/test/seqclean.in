I am looking for a clinic. I would like something that offers flu shots.
Yes, could you make an appointment for 1 at 10:00 on Saturday?
My name is Jason Bourne and number is 2103594867.
Thank you so much for your help!
Thanks, you too. Goodbye.
Hi, can you find me a clinic that offers moderna covid shots?
Are there any clinic that offer novaform?
Yes, the one in the east please.
Yes, please book it for 2 people at 13:45 on monday.
No thank you that will be all.
That was all thank you.
I am looking for a covid shot clinic in the north side of town
How about the south side of town?
Does it offer moderna?
book it for 3 people on tuesday at 11:00 am.
My name is Michael Jackson and number is 7103594862.
No that is all I need. Thank you for your time.
i am looking for a clinic in the west that offers covid shots
Any brand would be fine. Could I get the phone number for your recommendation?
Sounds good, could I get that phone number?
Yes. Can you book an appointment for me?
i want to book it for 2 people at 9:00 am on saturday.
My name is Mickey Salvador and number is 2503194867.
That is all I need to know. Thanks, good bye.
hi, I am in east san jose and need a clinic for covid shot.
I'd really prefer pfizer. Which one do you recommend?
Yes, book it for 3 people at 10:00 am on tuesday.
My name is Novak Djokivic and number is 2903514867.
How about for 11:00 am on tuesday?
Great. I am all set then. Have a nice day. Bye.
Hi there! I need a clinic in the east offers flu shots.
No, I don't have a preference, but I need the phone number and postal code.
No, right now, I just need the info. Thanks!
That is all. thanks
I am looking for a clinic named Eastern Health.
I would like to book an appointment for it. Can you help me with that?
i need flu shots.
I need it for Thursday at 11:00 am.
for 4 people.
I am Brad White and number is 2301594867.
that is it thank you.
I am looking for a clinic. I would like something that offers covid shots.
Yes, could you book an appointment for 2 at 13:00 on friday?
yes, pfizer please.
My name is Sheila Brown and number is 8103594267.
Thank you so much for your help!
I am looking for Riverside clinic.
Yes, I would like to book it on Tuesday at 13:00 and get a reference number.
My name is Ronald Sanchez and number is 6103594827.
Can you try for Wednesday at 11:30?
flu vaccine
3 people
Thanks for everything you have been great!
No that is all I needed. Thank you ever so kindly.
I'm looking for an flu shot clinic in the centre if you could help me.
No, not really. Could you suggest something?
Yes, I would like both the phone number and the postcode please.
that is it for today. good day thank you
Hi, I'd like to find a clinic for covid shots in the south?
The Lucky Star sounds good. That's all I need.
Can you also tell me the postcode?
Thanks for your help. Good bye.
I want to find a clinic in the west that offers flu shots. Can i also have the address, phone number and its zip code?
thank you goodbye
Where's a good place to get covid shots in centre sunnyvale.
How about in the south?
I would prefer the pfizer.
Yes please make a booking for 2 people at 12:30 on monday.
My name is Danelle Stroup and number is 4103592867.
That is all I needed.
I want a clinic in the north of Cambridge that offers covid shots.
Please book an appointment for 4 at the one that offers novaform at 11:30 on Sunday and give me the reference number.
My name is Dennis Scoma and number is 2130594867.
No, thank you. That will be all for today. Thanks.
I am looking for a clinic to get covid shot in the east. Phone #, postcode and address please.
I would prefer pfizer but moderna would be good as well. I will need the phone number and postcode of the clinic. Thanks!
Thanks for helping me with this. I appreciate it! Goodbye!
I need a clinic to get covid shots.
could you provide details for something that offers pfizer?
No, I have no preference. Just pick one. That will be fine.
Perfect. please book it for 1 person at 9:00 am on tuesday.
My name is Eric Emmons and number is 5103294867.
Nope, that's it. Thanks!
I would like to book a clinic in north for flu shots.
No, I don't really care. Just something in the north that offers flu shots.
Great!! Let get a booking for 2 people at 9:30 am on thursday.
My name is Claude Spell and number is 2703594861.
No, that will be all. Thanks, goodbye.
Can you help me book an appointment at a clinic for covid shots that's in the centre of town?
I need moderna, please.
That sounds good. Please book me an appointment for 3 on Tuesday at 12:45.
My name is Bob Manoit and number is 2103594867.
Can we try the same day for 11:45 please? And can I have the reference number as well, thanks.
That will be all. Thank you for your help!
Thanks, take care buddy!
I am looking for a clinic that offers covid shots and in the south part of town.
Let's go with Pfizer.
address and phone number please.
Thank you, goodbye.
I'm looking to get flu shots around the centre of town, can you find me some clinics?
Hmm... let me think. Let's try maybe somewhere in the south instead.
not really, what would you recommend? Can I please have the address and postcode.
No, that's all. Thanks.
I am looking for a particular clinic. Its name is called advantage clinic.
Yes, and can you book an appointment for 2 at 14:00 on Monday, please.
My name is Christine Smart and number is 9103514867.
Hi, can you find me an clinic that offers novaform covid shots?
Are there any clinic that offers moderna shots?
Yes, the one in the east please.
Yes, please book it for 2 people at 13:45 on wednesday.
My name is Susan Rodriguez and number is 2603549817.
No thank you that will be all.
That was all thank you.
I'm looking for a clinic to get covid shots. Moderna vaccine in preferred.
Yes, the clinic should be in the east.
nope, that's all I need for now. Thanks.
Thanks you too Goodbye
I am looking for a clinic.
Not sure of that, I am looking for a clinic named Nature.
I would like to book an appointment for 3 people on tuesday at 10:00
My name is Afsoun Shields and number is 7603594812.
Yes, a different clinic in the same area and one that offers covid shots.
vaccine brand doesn't matter. anyone will work.
That is all. Thank you!
I am looking for a covid shot clinic in the north side of town
How about south side of town?
Does it offer pfizer?
book it for 2 people at 15:00 pm on monday.
My name is Crystal Williams and number is 2803594167.
No that is all I need. Thank you for your time.
Hi, what can you tell me about the Civic Medical centre?
Yes for 1 at 09:30 on Thursday please. I need the reference number then too.
My name is Monica McCarthy and number is 2105394867.
Thank you so much.
No that is it. Thank you so much.
Thank you for your assistance.
Please find me a place to get covid shots and in the centre.
I am looking for moderna vaccine.
Ok great. Can you book an appointment there for Thursday for 1 person at 13:00 please?
My name is Susan Pommers and number is 2168594307.
Thank you for your help.
That is everything. Thank you so much for your help.
I'm looking for a particular clinic called Modern. Please give me information on that clinic.
Yes, I'd like to book an appointment for 1 at 09:15 on Wednesday. Can you do that?
My name is Ernesto Ruffino and number is 2109534867.
No. I think I am all set. Thanks.
Hey, I'm up south can you give me a clinic to get flu shots?
Can I have the address and telephone number?
THAT WILL BE FINE.
Can you assist me in finding me a clinic in east gilroy where I can get covid shots.
I'd like moderna.
Can you make me an appointment for Wednesday please for 3 people?
We prefer 9:30am. Please book for 3 on Wednesday and provide the reference number.
My name is Mike Morley and number is 3501794862.
Thanks so much!
No, that will be all. Thank you!
Thanks for all the help.
I need a clinic with flu shots. I am in south part of the town.
Yes, please book an appointment it for Thursday at 14:00 hours for 2 people.
My name is Bruce Longbottom and number is 3015794862.
That is all I need.
I live in the centre of town and want to see if I can find a clinic to get flu shots for my family.
That sounds like it might work. Could you check availability for 2 people this week in the morning?
I am unsure at this time.
Can you tell me what the address is?
Does this clinic offer covid shots as well?
You have been very helpful. I have everything I need, thanks. Bye
I am looking for information in Burlingame. I need a clinic.
covid shots.
are there a flu clinics in centre?
nope, you choose one for me. Can I please have the phone number?
No thank you that will be all
No thank you! Thanks for your help! Bye!
I'm looking for a particular clinic. Its name is called Health First.
I'd like to to book an appointment for 4 people at 13:15 on tuesday
My name is Jacob Willis and number is 5793014862.
Not at this time. Thank you.
I'm looking for a local clinic in the centre that offers covid shots.
I need the address, postcode and the phone number.
could you please book an appointment for 5 people and get me the reference number.
friday at 16:00 pm.
I am John Hemsworth and number is 4873012732.
No, this is all I will need. Thank you.
My husband and I are seeking to get covid shots and looking for a clinic.
I would prefer west.
yes moderna please.
Yes, please! book it at 8:00 am on Monday for the two of us!
I am Megan Fernandez and my phone number is 4087317239.
Thanks so much. That is all I need for today. Bye.
I need a clinic to get flu shots for 2 on monday at 11:30.
east please.
I am Kevin Fu and my phone number is 7298047239.
Nope, that is everything! Thanks!
I'm looking for a place to get covid shots in the south of town.
novaform please.
yes please.
could you please share address and phone number?
No, that is all. Thank you! Goodbye!
I want general information on clinics to go in the west area of town.
I need one that offers moderna covid shots? I'll need the address, phone and postcode please.
Okay. I need to book an appointment for 6 on monday at 9:00 am please.
I am Kevin Moyer and my phone number is 7227801339.
No. You took care of everything. Thanks. Goodbye.
i want information about a clinic in the centre of the town that offers covid shots?
I don't need an appointment right now, but I do need the postcode and phone number for that clinic.
cool, thanks. i don't need anything more. see ya!
I'm looking to covid shots for my family. Are there facilities that cater to that?
Do you have anything like that in the centre?
What about south part of the town?
Yes please, and I'm going to need the address.
What is their address?
Yeah, can you book appointment for 4 on monday for me?
8:30 am in the morning.
I am Brandon Southee and my phone number is 2787201339.
Thanks a bunch! I couldn't have done it without you. Bye.
Are there any clinics in the east part of town that offer covid shots?
I want moderna.
address?
could you book it on wednesday at 13:00
Just one person please and I need a reference number
Mike Atherton and 9012787339
Great, thank you. That is all I need.
Thanks again for your help.
I need flu shot appointment in east san bruno
I'd like it on Thursday 15:00 for 1 person.
No hold off on booking for now. Can you share address and phone number?
Nope, that was all I needed. Thank you very much!
Hello, I am looking for a clinic in moraga. I believe it is called Golden Eagle.
Can you book me an appointment for 11:00 on Friday?
Actually, for 4, please.
I am Michelle Strauss and contact number is 2790187339
Thanks for the service, good day.
Hi, I'm looking for a clinic to get covid shots. I prefer pfizer.
I am in the west part of the town.
Okay, please book that for 3 people at 10:00 am on Friday.
Sure, I am Jim Wasser and 6347289046 is my phone number.
Thank you so much for your help.
I am looking for a clinic to get flu shots in the north of the city.
yes please.
I think that would be fine.
I would like it book for Saturday for 5 people at 9:00. Can I have the booking reference please.
Sure, I am Joey Bela and phone is 4890726346.
Thank you, I think that is all I need for today.
I need a place to get covid shots in centre of the city.
I need to get the address please.
Great. That was it. Thank you.
No thanks again for all of your help
I need to reserve an appointment at Northside Bay clinic, can you help me?
I need the booking for Tuesday.
There will be 3 of us and 9:45 would be great.
Sure, I am Jim Wyatt and phone is 4726389046.
Thank you so much.
No, that will be all. Thanks.
I am looking for clinic in east milpitas
I need to get family vaccinated for flu shots. can you book it for me please?
Can you book this for 3 people on tuesday morning at 8:00? Reference please?
I am Mikah Schneider and phone is 3813456770.
No, that's all I needed today. Thanks for your help, it's much appreciated.
I need a clinic at 10:00 on wednesday for 2 people.
covid shots.
novaform
west and share the reference number please.
I'm Nancy Drew and my phone is 4088044800
Thank you. I think that's all I need.
I need to get pfizer covid shots tuesday at 13:30. Can you book it for me?
centre please
Yes that is perfect. Can you book that and give me a reference number?
my name is Ron Balboa and my number is 5102209382
No, I think that's going to be all I needed. Thanks. Have a good day.
I would like to find a clinic to get covid shots. Do you have any suggestions?
Do they offer moderna vaccine?
Yes, please book for 5 people at 9:00 am on Thursday.
How about wednesday
Wednesday at 11:00 am?
Mickey Wong and 6483917234
Thank you so much. Good bye
I am looking for a place to get flu shots in the centre.
Is there a clinic located in the north then?
Yes please. I would need name and phone number.
No, that's all I needed. Thank you!
I am looking for a clinic to get pfizer covid shot.
I'd like it to be in the north.
Yes, please book it for 1 on monday at 13:00 pm.
myself Jack Brown and my phone is 9257364729
No that is all. Thanks for your help!
I would like to find out information about a clinic called Parkmont clinic.
Yes please. It'll be for 4 people on monday at 9:00 am.
Heather Rose and 4285432345
Thank you. Can I get a reference number for the booking?
That's all for now. Thank you for all your help.
I need a clinic that offers covid shots, thank you
I'm looking for something that offers pfizer actually.
Yes please book the Archway clinic.
I want appointment on tuesday at 8:00 am for three people.
My name is James Bond and number is 1112347007
Thank you very much! That's all I need today!
Hi, I need moderna covid shots for a family of 3 in the south of town.
Yes please. May I book an appointment 16:00 on Wednesday and get a reference number please?
Yes, for 3 people please. Could I also get a reference number?
Helena Yates and my number is 8635637482.
thank you, that is all I need.
We need covid shot clinic on the west side of town.
I want novaform.
Yeah, can you check the south please?
What a bummer. Find me a clinic that offers moderna food on the West please.
yes please. I will need their phone number and address.
Thanks so much. That will be all the information I need today. Bye.
I am seeking a clinic in napa
I'm actually looking to book a specific clinic. It is called the Napa Valley clinic.
I'd like to book it on Wednesday, it will be at 9:00 am in the morning, and it will be 4 people.
My name is Charles Knight and my number is 5109752347.
No thank you. Goodbye.
I'm looking for a clinic in the east. I need to get covid shots.
yes very much. please book it for me for three people.
I want it on Tuesday noon at 1:00 pm.
Can you see if they have 3:00 pm available on same day?
yes, my name is Jennifer Capriati and phone is 8253467823.
Awesome, thanks! I think that takes care of everything I needed.
I'm looking for a clinic that offers covid shots.
I would like a clinic in the centre and it should offer novaform.
Yes, that would be fine. Could you suggest one?
Okay may I please have the phone number?
Thank you very much.
No thank you that will be all
Let's find me an clinic to get flu shots in the west
How about you pick one and make an appointment for 5 on thursday at 11:45, and I'll need that reference number too.
yes, I am Steve Trush and phone is 9256347283.
Thank you very much.
No, that's all. Good bye
I would like to get covid shots in the centre of town.
How about Moderna?
great. I'd like to book an appointment for 1 at 16:15 on Tuesday.
sure, I am Susan Sarandon and my phone is 9427442491.
Nope that's all I needed! Thanks!
Can you help me find a clinic to get pfizer covid shots?
west side please.
It doesn't matter. Like I told you, you can pick one for me.
Okay I would like to make a booking for 2 people at 10:00 am on wednesday.
Yes, of course.
Brad Hogg and my number is 2245718243
That is all. Thanks for your time.
I don't know if this is possible but can you please get me some information on the Nominal Healthcare clinic?
Ah, I just wanted to know it offers covid shots. if yes, could you book me an appointment?
3:00 pm on friday.
No, I would like it for 4 people.
Julia Clobber and my phone number is 1082734684.
Great, thanks so much, that's all I need! Have a great day!
I need a place to get covid shots.
In the centre, please.
i want novaform.
I am fine with pfizer also. please pick one for me.
Yes please. Can you book that for 5 people on Saturday at 13:30?
Vicky Henson and my number is 8724857489.
Okay, thank you.
If Saturday fails, try Monday and provide the reference number.
Nope, that's all for now. Thank you for all your help.
I am seeking a covid clinic in santa monica
I need a clinic that offers covid shots on friday anytime after 13:00
No area doesn't matter and i prefer moderna. Can you pick one, and make me a booking for three people? I need a reference number too!
my name is Nick Morley and phone is 7394727393.
Thank you. Thats all I need for today.
Hello, I'm looking for a clinic to get flu shots. I'd like a place in east part of the town, please.
That's fine. Can you book it for 2 people at 11:30 am on Thursday?
Dillon Straight and phone number is 7234522124
No thank you. I appreciate all your help.
I need a covid shot appointment on Tuesday for 3 at 11:15am.
I prefer moderna in the east part of the town.
yes please.
My name is John Cusack and my number is 8652312234. I will need the reference number.
No thank you. That's all I need for now.
Hi, I am looking to get flu shots for the family and could use some help with a particular clinic.
I'm looking for Milton Park clinic.
Can I get the address, phone and postcode please?
Great! Thank you! That's all I needed.
Hi, can you recommend a covid shot clinic for me?
I'd like it to be in the centre please.
it doesn't really matter as long as it is located in the centre. I need phone number and address.
Great, thank you. That is all that I need.
Hi, I want a clinic for moderna covid shots located in the south part of town.
address and postcode please
No, thank you, goodbye.
I would like clinic to get covid shots in east santa clara.
I do not have a preference of vaccine. Please make a suggestion.
Can I get the address and phone number?
No, thank you. Good bye.
Hi! Would you please help me find a clinic for covid shots?
It needs to be in centre of the town
I prefer moderna. Please pick one and get me a booking for 3 people on Friday at 8:30 am.
my name is Jessica Pommer and phone number is 9254327586
No thanks, that will be everything.
Thank you for your time goodbye!
