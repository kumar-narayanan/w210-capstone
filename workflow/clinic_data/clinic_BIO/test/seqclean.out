O O O O O O O O O O O O B-type O
O O O O O O O B-bookpeople O B-booktime O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O
O O O O
O O O O O B-type O O O O B-area O O O
O O O B-area O O O
O O O B-vaccinebrand
O O O B-bookpeople O O B-bookday O B-booktime O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O O
O O O O O O O O B-area O O B-type O
O O O O O O O O O O O O O O
O O O O O O O O
O O O O O O O O
O O O O O O B-bookpeople O O B-booktime O O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O
O O O O B-area O O O O O O O B-type O
O O O B-vaccinebrand O O O O O
O O O O B-bookpeople O O B-booktime O O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O B-booktime O O B-bookday
O O O O O O O O O O O
O O O O O O O O O O B-type O
O O O O O O O O O O O O O O O
O O O O O O O O O
O O O O
O O O O O O O B-name I-name
O O O O O O O O O O O O O O O
O O B-type O
O O O O B-bookday O B-booktime O
O B-bookpeople O
O O B-customername I-customername O O O B-customer-phonenumber
O O O O O
O O O O O O O O O O O O B-type O
O O O O O O O O O B-booktime O O
O B-vaccinebrand O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O
O O O O B-type O O O O B-area O O O O O
O O O O O O O
O O O O O O O O O O O O
O O O O O O O O O
O O O O O O O O B-type O O O B-area
O B-name I-name O O O O O O
O O O O O O O
O O O O O O
O O O O O O O O B-area O O B-type O O O O O O O O O O O O O
O O O
O O O O O O B-type O O B-area O
O O O O B-area
O O O O B-vaccinebrand
O O O O O O B-bookpeople O O B-booktime O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O
O O O O O O B-area O O O O B-type O
O O O O O B-bookpeople O O O O O B-vaccinebrand O B-booktime O B-bookday O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O
O O O O O O O O B-type O O O B-area O O O O O O
O O O B-vaccinebrand O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O
O O O O O O O O B-area O B-type O
O O O O O O O O O O O O O O
O O O O O O B-bookpeople O O B-booktime O O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O
O O O O O O O O O O O B-type O O O O B-area O O
O O B-vaccinebrand O
O O O O O O O O O B-bookpeople O B-bookday O B-booktime
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O B-booktime O O O O O O O O O O O
O O O O O O O O O
O O O O
O O O O O O O O B-type O O O O B-area O O O
O O O B-vaccinebrand
O O O O O
O O O
O O O O B-type O O O B-area O O O O O O O O
O O O O O O O O O O B-area O
O O O O O O O O O O O O O O
O O O O
O O O O O O O O O O O B-name O
O O O O O O O O B-bookpeople O B-booktime O B-bookday O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O B-type O B-vaccinebrand O O O
O O O O O O O B-area
O O O O O O O O
O O O O
O O O O O O
O O O O O O O O O O O B-name
O O O O O O O O B-bookpeople O O B-bookday O B-booktime
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O O O O O
O O O O O O O
O O O O O
O O O O O B-type O O O O B-area O O O
O O B-area O O O
O O O B-vaccinebrand
O O O B-bookpeople O O B-booktime O O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O O
O O O O O O O O B-name I-name I-name
O O B-bookpeople O B-booktime O B-bookday O O O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O
O O O O O O O O
O O O O O
O O O O O O O B-type O O O O B-area
O O O O B-vaccinebrand O
O O O O O O O O O B-bookday O B-bookpeople O O B-booktime O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O
O O O O O O O O O O
O O O B-area O O O O O O O O B-type O
O O O O O O O O
O O O O
O O O O O O O O O O B-area O O O O O B-type O
O O B-vaccinebrand
O O O O O O O B-bookday O O B-bookpeople O
O O B-booktime O O O O O O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O
O O O O O O O
O O O O O
O O O O O B-type O O O O B-area O O O O
O O O O O O O B-bookday O B-booktime O O B-bookpeople O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O
O O O O O O O O O O O
B-type O
O O O B-type O O O
O O O O B-type O O O O O O O O
O O O O O O O
O O O O O O O O
O O O O O O O O O O B-name I-name
O O O O O O O O B-bookpeople O O B-booktime O B-bookday
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O
O O O O O O O O B-area O O B-type O
O O O O O O O O O
O O O O O O O B-bookpeople O O O O O O O
B-bookday O B-booktime O
O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O
O O O O O O B-type O O B-bookpeople O B-bookday O B-booktime
B-area O
O O B-customername I-customername O O O O O B-customer-phonenumber
O O O O O
O O O O O O O B-type O O O B-area O O
B-vaccinebrand O
O O
O O O O O O O O
O O O O O O O
O O O O O O O O O O B-area O O O
O O O O O B-vaccinebrand B-type O O O O O O O O O
O O O O O O O O B-bookpeople O B-bookday O B-booktime O O
O O B-customername I-customername O O O O O B-customer-phonenumber
O O O O O O O O
O O O O O O O O B-area O O O O O B-type O
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O
O O O B-type O O O O O O O O O O O
O O O O O O O O B-area
O O B-area O O O O
O O O O O O O O O
O O O O
O O O O O O B-bookpeople O B-bookday O O
B-booktime O O O O
O O B-customername I-customername O O O O O B-customer-phonenumber
O O O O O O O O O O O
O O B-type O O O B-area O O
O O O O B-bookday B-booktime O B-bookpeople O
O O O O O O O O O O O O O O
O O O O O O O O O O
O O O O O O O O O O O O O O B-name I-name
O O O O O O O B-booktime O B-bookday
O O B-bookpeople O
O O B-customername I-customername O O O O B-customer-phonenumber
O O O O O O
O O O O O O O O B-type O O O B-area O O O
O O
O O O O O O
O O O O O O B-bookday O B-bookpeople O O B-booktime O O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O O
O O O O O O B-type O O B-area O O O
O O O O O O O
O O O O O O
O O O O O O O O
O O O O O O O B-name I-name O O O O O
O O O O O B-bookday
O O O B-bookpeople O O O B-booktime O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O
O O O O O O
O O O O O O B-area O
O O O O O O O B-type O O O O O O O O
O O O O O B-bookpeople O O B-bookday O O B-booktime O O
O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O O O O O O
O O O O B-vaccinebrand B-type O B-bookday O B-booktime O O O O O O
B-area O
O O O O O O O O O O O O O O
O O O B-customername I-customername O O O O B-customer-phonenumber
O O O O O O O O O O O O O O O
O O O O O O O O O B-type O O O O O O
O O O B-vaccinebrand O
O O O O B-bookpeople O O B-booktime O O B-bookday
O O B-bookday
O O B-booktime O
B-customername I-customername O B-customer-phonenumber
O O O O O O
O O O O O O O O B-type O O O B-area
O O O O O O O B-area O
O O O O O O O O O
O O O O O O O
O O O O O O O O O O O B-name O
O O O O O B-bookpeople O O B-bookday O B-booktime O
B-customername I-customername O B-customer-phonenumber
O O O O O O O O O O O
O O O O O O O O O O
O O B-type O O O O B-area O O O
O O B-vaccinebrand
O O O O O B-area O
O O O O O O O O O B-vaccinebrand O O O B-area O
O O O O O O O O O O
O O O O O O O O O O O O O
O O O O O O O
O O O O O O O O O O O O B-name I-name O
O O O O O O B-bookday O O O O B-booktime O O O O O O O O B-bookpeople O
O O O B-customername I-customername O O O O B-customer-phonenumber
O O O O
O O O O O O O B-type O
O O O O O O O B-area O O O O B-vaccinebrand
O O O O O O O O O
O O O O O O O O
O O O O
O O O O O O O
O O O O O O O B-type O O O B-area
O O O O O O O O O O B-bookpeople O B-bookday O B-booktime O O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O
O O O O O
O O O O O B-type O O O B-area O O
O O B-vaccinebrand
O O O O O O O O B-bookpeople O B-booktime O B-bookday
O O O B-customername I-customername O O O O B-customer-phonenumber
O O O O O O
O O O O O O O O O O O O O O O O O B-name I-name O
O O O O O O O O O O O O O O O O O O
B-booktime O O B-bookday
O O O O O O B-bookpeople O
B-customername I-customername O O O O O B-customer-phonenumber
O O O O O O O O O O O O
O O O O O O B-type O
O O B-area O
O O B-vaccinebrand
O O O O B-vaccinebrand O O O O O O
O O O O O O O B-bookpeople O O B-bookday O B-booktime
B-customername I-customername O O O O B-customer-phonenumber
O O O
O O O O O O O O O O
O O O O O O O O O O O
O O O O B-type O O O O
O O O O O O B-type O O B-bookday O O B-booktime
O O O O O O O B-vaccinebrand O O O O O O O O O O B-bookpeople O O O O O O O
O O O B-customername I-customername O O O B-customer-phonenumber
O O O O O O O O
O O O O O O O O B-type O O O O O O B-area O O O O O
O O O O O O O B-bookpeople O O B-booktime O O B-bookday
B-customername I-customername O O O O B-customer-phonenumber
O O O O O O O O
O O O B-type O O O B-bookday O B-bookpeople O B-booktime
O O B-vaccinebrand O O B-area O O O O
O O
O O O B-customername I-customername O O O O B-customer-phonenumber O O O O O O
O O O O O O O O O
O O O O O O B-type O O O O O O O O O O O O O
O O O B-name I-name O
O O O O O O O O O
O O O O O O O
O O O O O B-type O O O O
O O O O O O O B-area O
O O O O O O O O O O O O B-area O O O O O O
O O O O O O O O O
O O O O O O B-vaccinebrand B-type O O O O B-area O O O
O O O O
O O O O
O O O O O O O O O B-area O O
O O O O O O O O O O O O
O O O O O O O O
O O O O O
O O O O O O O O O O B-type O
O O O O O B-area O O O
O O B-vaccinebrand O O O O O O O O O B-bookpeople O O B-bookday O B-booktime O
O O O B-customername I-customername O O O O B-customer-phonenumber
O O O O O O
O O O O O O
